apps/include/netutils/libcoap/address.h

```diff
@@ -0,0 +1,164 @@
+/* address.h -- representation of network addresses
```

Comment:
Dear. Issgood

1) We found requirements that user wants to use CoAP independently. 
   Currently, er-coap is included on wakaama and libcoap-4.1.1 is included on Iotivity.
   If user wants to deploy IoT devices through CoAP, user should select either wakaama or Iotivity.

2) There is no way to verify CoAP stack independently.

CoAP is one of popular protocol in IoT world.
So, we thought that TizenRT should support CoAP library.

We've selected libcoap since, er-coap is based on Contiki OS and not considering network layer
libcoap is based on POSIX API and it is more suitable for TizenRT

Thanks & Regards